cmd/price_check/handler_test.go

```diff
@@ -0,0 +1,117 @@
+// E2E test of endpoints
+package main
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"os"
+	"testing"
+
+	"github.com/btc-price/cmd/price_check/handler"
+	"github.com/btc-price/internal/btcpriceservice"
+	"github.com/btc-price/internal/coingeckoclient"
+	"github.com/btc-price/internal/emailsender"
+	"github.com/btc-price/internal/emailstorage"
+	"github.com/stretchr/testify/assert"
+	"go.uber.org/zap"
+)
+
+func makeTestRouter(cfg Config) http.Handler {
+	logger, _ := zap.NewProduction()
```

Comment:
```suggestion
	logger, _ := zap.NewDevelopment()
```

Зараз в проєкті немає логів рівня DebugLevel, але краще відразу закласти це, ніж потім бігати і міняти